pkg/controller/core/clusterqueue_controller_test.go

```diff
@@ -499,11 +500,13 @@ func TestClusterQueuePendingWorkloadsStatus(t *testing.T) {
 		queueVisibilityUpdateInterval        time.Duration
 		queueVisibilityClusterQueuesMaxCount int32
 		wantPendingWorkloadsStatus           *kueue.ClusterQueuePendingWorkloadsStatus
+		enableQueueVisibility                bool
```

Comment:
nit: add a test case where `enableQueueVisibility ` is false, but the configuration is provided

Reply:
added https://github.com/kubernetes-sigs/kueue/pull/1139/commits/a868d19bd40e441fde892df247fdf8a5f55419ee